{% if debug %}
    <h2>Stack trace</h2>

    <ol class='stack-trace'>
    {% for t in exception.trace %}
        <li>
            <span># {{ loop.index }}. </span>
            {% if t.class is defined %}
                {{ t.class|default()|split("\\")[0:-1]|join("\\") -}}\<strong>{{- t.class|default()|split("\\")|last -}}</strong>::
            {% endif %}
            {{ t.function|default() }}()<br>
            <em class='dim'>{{ t.file|default() }} # line <strong>{{ t.line|default() }}</strong></em>
        </li>
    {% endfor %}
    </ol>

{% else %}

    <p>Debug is not available, and/or you are not logged on. To see a full backtrace of the Exception, do one of the following: </p>

    <ul>
        <li>Set <tt>debug: true</tt> in <tt>config.yml</tt>, and make sure you're logged in to the Bolt Backend.</li>
        <li>Set both <tt>debug: true</tt> and <tt>debug_show_loggedoff: true</tt> in <tt>config.yml</tt>.
    </ul>

{% endif %}
